src/index.js

```diff
@@ -1,38 +1,219 @@
+          if ((this.tasksList[i].description.trim().toString()) === ((cbTemp).trim().toString())) {
+            this.tasksList[i].completed = !(this.tasksList[i].completed);
+            ToDoList.updateLocalList(this.tasksList);
+          }
+        }
+        /*
+
+        (this.tasksList).forEach(task =>{
+          const cbTemp = cb.parentElement.lastElementChild.innerHTML;
+        if ((((task.description).trim().toString()) === ((cbTemp).trim().toString()))){
+          task.completed = !(task.completed);
+          this.updateLocalList(this.tasksList);
+        }
+      }); */
+      });
+    });
+
+    // find the delete icons here
+    const deleteIcons = document.querySelectorAll('.fa-trash-alt');
+    deleteIcons.forEach((di) => {
+      di.addEventListener('click', (e) => {
+        ToDoList.deleteTask(e.target);
+      });
+    });
+  }
+}
+
+const myList = new ToDoList();
+
+const refresh = () => {
+  myList.tasksList = ToDoList.getLocalList();
+  myList.tasksList.forEach((task) => {
+    const taskElement = document.createElement('div');
+    taskElement.className = 'tasks';
+    taskElement.innerHTML = `
   <div class="task-wrapper">
-  <input type="checkbox" name="completed" class="completed">
-  <span>${task} </span>
+  <input type="checkbox" name="completed" class="status">
+  <span>${task.description} </span>
   </div>
   <i class="fas fa-ellipsis-v"></i>
   <i class="fas fa-trash-alt"></i>
   `;
-  tasksSection.appendChild(taskElement);
-}
-tasksList.forEach((taskItem) => {
-  component(taskItem.task);
+    tasksSection.appendChild(taskElement);
+
+    const checkBoxes = document.querySelectorAll('.status');
+
+    checkBoxes.forEach((cb) => {
+      cb.addEventListener('click', (e) => {
+        console.log(e.target.nextElementSibling.textContent);
+        cb.parentElement.parentElement.classList.toggle('task-bg');
+        // cb.parentElement.lastElementChild.classList.toggle('task-completed');
+        cb.nextElementSibling.classList.toggle('task-completed');
+        cb.parentElement.parentElement.lastElementChild.classList.toggle('delete-task-icon');
+        cb.parentElement.parentElement.lastElementChild.previousElementSibling.classList.toggle('edit-task-icon');
+
+        for (let i = 0; i < myList.tasksList.length; i += 1) {
+          const cbTemp = cb.parentElement.lastElementChild.innerHTML;
+          if ((myList.tasksList[i].description.trim().toString()) === (cbTemp.trim().toString())) {
+            myList.tasksList[i].completed = !(myList.tasksList[i].completed);
+            ToDoList.updateLocalList(myList.tasksList);
+          }
+        }
+        /*
+
+        (myList.tasksList).forEach(task =>{
+           const cbTemp = cb.parentElement.lastElementChild.innerHTML;
+      if ((((task.description).trim().toString()) === ((cbTemp).trim().toString()))){
+        task.completed = !(task.completed);
+        ToDoList.updateLocalList(myList.tasksList);
+      }
+     }); */
```

Comment:
### Optional
- [ ] It would be better if we remove all commented piece of code, this makes our code base looks messy and difficult to understand